src/adagrams.js

```diff
@@ -1,15 +1,135 @@
+// import { random } from "core-js/core/number";
+
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const LETTER_POOL = {
+    A: 9,
+    B: 2,
+    C: 2,
+    D: 4,
+    E: 12,
+    F: 2,
+    G: 3,
+    H: 2,
+    I: 9,
+    J: 1,
+    K: 1,
+    L: 4,
+    M: 2,
+    N: 6,
+    O: 8,
+    P: 2,
+    Q: 1,
+    R: 6,
+    S: 4,
+    T: 6,
+    U: 4,
+    V: 2,
+    W: 2,
+    X: 1,
+    Y: 2,
+    Z: 1,
+  };
+
+  const copy = { ...LETTER_POOL }
+  const tenLetters = [];
+
+  while (tenLetters.length < 10) {
+    let randomNum = Math.floor(Math.random() * Object.keys(copy).length);
```

Comment:
Create a list of duplicate letters and draw from this list?

Reply:
Yep, that's one option! Otherwise we can dig into how to weight our choices to favor letters there should be more of (someone shared [this article](https://dev.to/trekhleb/weighted-random-algorithm-in-javascript-1pdc) that talks about an approach for weighting the choices).